dev/deploy-to-container/start.sh

```diff
@@ -12,28 +12,9 @@ echo "Running Datatracker checks..."
 
 # Migrate, adjusting to what the current state of the underlying database might be:
 WORKSPACEDIR=.
-if ietf/manage.py showmigrations | grep "\[ \] 0003_pause_to_change_use_tz"; then
-    if grep "USE_TZ" $WORKSPACEDIR/ietf/settings_local.py; then
-        cat $WORKSPACEDIR/ietf/settings_local.py | sed 's/USE_TZ.*$/USE_TZ = False/' > /tmp/settings_local.py && mv /tmp/settings_local.py $WORKSPACEDIR/ietf/settings_local.py
-    else
-        echo "USE_TZ = False" >> $WORKSPACEDIR/ietf/settings_local.py
-    fi
-    echo "Running Datatracker migrations with USE_TZ = False..."
-    # This is expected to exit non-zero at the pause
-    /usr/local/bin/python $WORKSPACEDIR/ietf/manage.py migrate --settings=settings_local || true
-    cat $WORKSPACEDIR/ietf/settings_local.py | sed 's/USE_TZ.*$/USE_TZ = True/' > /tmp/settings_local.py && mv /tmp/settings_local.py $WORKSPACEDIR/ietf/settings_local.py
-    echo "Running Datatracker migrations with USE_TZ = True..."
-    /usr/local/bin/python $WORKSPACEDIR/ietf/manage.py migrate --settings=settings_local
 
-else
-    if grep "USE_TZ" $WORKSPACEDIR/ietf/settings_local.py; then
-        cat $WORKSPACEDIR/ietf/settings_local.py | sed 's/USE_TZ.*$/USE_TZ = True/' > /tmp/settings_local.py && mv /tmp/settings_local.py $WORKSPACEDIR/ietf/settings_local.py
-    else
-        echo "USE_TZ = True" >> $WORKSPACEDIR/ietf/settings_local.py
-    echo "Running Datatracker migrations..."
-    /usr/local/bin/python $WORKSPACEDIR/ietf/manage.py migrate --settings=settings_local
-    fi
-fi
+echo "Running Datatracker migrations..."
+/usr/local/bin/python $WORKSPACEDIR/ietf/manage.py migrate --settings=settings_local
```

Comment:
This is the only spot using `WORKSPACEDIR` in this script - everywhere else is just an explicit `./`. Get rid of the variable (or replace `./` everywhere)?